docker-compose.yml

```
@@ -64,9 +65,25 @@ services:
       - "9200:9200"
     mem_limit: 2g
 
-  # -- tools
   ralph:
     image: fundocker/ralph
+    ports:
+      - "${RALPH_RUNSERVER_PORT:-8100}:${RALPH_RUNSERVER_PORT:-8100}"
```

You may want to define a different default from `warren` service:

```suggestion
      - "${RALPH_RUNSERVER_PORT:-8200}:${RALPH_RUNSERVER_PORT:-8200}"
```